Formula/cockroach.rb

```diff
@@ -23,7 +23,7 @@ def install
     go_version = Formula["go"].installed_version.to_s.split(".")[0, 2].join(".")
     inreplace "src/github.com/cockroachdb/cockroach/.go-version",
               /^GOVERS = go.*/, "GOVERS = go#{go_version.gsub(".", "\\.")}.*"
-
+    ENV["CGO_LDFLAGS_ALLOW"] = "-u_je_zone_register"
```

Comment:
why isn't this handled in the Makefile?

Reply:
It was not originally required to build with Go version 1.8.3, which is what Cockroach's 1.1 branch was originally pinned to. A patch to allow building with newer Go versions (without the env var), has since been added, but is not in 1.1.6 as tagged.

Reply:
(added a comment and a TODO to clarify).